circuits/package.json

```diff
@@ -46,9 +46,9 @@
     },
     "dependencies": {
         "argparse": "^2.0.1",
-        "circomlib": "git://github.com/weijiekoh/circomlib.git#ac85e82c1914d47789e2032fb11ceb2cfdd38a2b",
+        "circomlib": "github:weijiekoh/circomlib#ac85e82c1914d47789e2032fb11ceb2cfdd38a2b",
```

Comment:
nit: another simplification would be`weijiekoh/circomlib#ac85e82c1914d47789e2032fb11ceb2cfdd38a2b` (see: https://docs.npmjs.com/cli/v9/configuring-npm/package-json#github-urls)